src/functions/sendCandidateResults/application/service/__tests__/fault-provider.spec.ts

```diff
@@ -1,21 +1,23 @@
 import {
-    DrivingFaults,
-    SeriousFaults,
-    TestData,
+  DrivingFaults,
+  SeriousFaults,
 } from '@dvsa/mes-test-schema/categories/common';
 import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
 import {
-    convertNumericFaultObjectToArray,
-    convertBooleanFaultObjectToArray,
-    getCompletedManoeuvres,
-    getVehicleChecksFault,
-    FaultProvider,
+  convertNumericFaultObjectToArray,
+  convertBooleanFaultObjectToArray,
+  getCompletedManoeuvres,
+  FaultProvider,
 } from '../fault-provider';
 import { Fault } from '../../../domain/fault';
 import { CompetencyOutcome } from '../../../domain/competency-outcome';
 import { Competencies } from '../../../domain/competencies';
+import * as catBFaultProvider from '../categories/B/fault-provider-cat-b';
+import * as catBEFaultProvider from '../categories/BE/fault-provider-cat-be';
 
 describe('fault-provider', () => {
+  const catB: string = 'B';
+  const catBE: string = 'BE';
```

Comment:
So the category in the test data I believe is B+E  not BE worth replacing throughout the code